lib/oli_web/live/delivery/student/discussions_live.ex

```diff
@@ -1029,12 +1029,20 @@ defmodule OliWeb.Delivery.Student.DiscussionsLive do
               "Page #{post.resource_numbering_index}"
 
             container_resource_id ->
-              container_with_numbering_index =
-                Map.get(ordered_containers_map, container_resource_id)
-                |> String.split(":")
-                |> hd()
-
-              "#{container_with_numbering_index}: Page #{post.resource_numbering_index}"
+              case Map.get(ordered_containers_map, container_resource_id) do
```

Comment:
Handle the case where the container_resource_id is the curriculum resource_id, which is not actually present in the new ordered_containers_map